about/3 lesson/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
```

Comment:
why did you create folder `3 lesson` in about directory for your home task ?). You should to do that in the root index.html file. `./index.html`. Delete `lesson 3` folder, move code to `./index.html` file.

Reply:
moved code to ./index.html file.